spec/spec_helper.rb

```diff
@@ -0,0 +1,105 @@
+require 'capybara/rspec'
+#require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+
+options = { js_errors: false}
+Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(app, options)
+end
+
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+RSpec.configure do |config|
+  # rspec-expectations config goes here. You can use an alternate
```

Comment:
I am confused by something here.  I would expect a statement like  config.include FactoryBot::Syntax::Methods which is what allows you to invoke methods like create and build within your controller tests without having to say FactoryBot.create or FactoryBot.build .  How is this working if you don't have the config.include statement?  It does seem to be working, but I don't understand why.

Reply:
That line of code is in the Rails Helper file.